### DISCUSSION TITLE
Add a permission to tp players with tp disabled

### DISCUSSION THREAD

**Discussion by thiagogebrimm** [EssentialsX/Essentials#4971] [Ideas and Feature Suggestions] [answered]
It would help a lot, if the player disables TP we need to use /minecraft:tp to go to them

**Comment by mdcfe** [comment, marked as answer]
`/tpo` exists for this exact purpose.

**Reply by thiagogebrimm** [reply]
But don't you think that permission for that would be more practical?

**Reply by osfanbuff63** [reply]
`essentials.tpo` exists for that purpose I think.
